contracts/multisig-prover/src/encoding/mod.rs

```diff
@@ -70,7 +70,7 @@ where
     let recovery_transform = match encoder {
         Encoder::Abi => add_27,
         Encoder::Bcs => no_op,
-        Encoder::StellarXdr => no_op,
+        _ => panic!("unsupported encoder"),
```

Comment:
is it not being called by stellar?

Reply:
yes, stellar use ed225519, this function is for ecdsa recover. probably the function itself needs some factor.